README.md

```diff
@@ -42,32 +42,32 @@ Spans:
 In your JavaScript code, add instrumentation to the operations to be tracked. This is composed primarily of using "spans" around operations of interest and adding log statements to capture useful data relevant to those operations.
 
 ```js
-var http = require('http');
-var opentracing = require('opentracing');
+const http = require('http');
+const opentracing = require('opentracing');
 
 // NOTE: the default OpenTracing tracer does not record any tracing information.
 // Replace this line with the tracer implementation of your choice.
-var tracer = new opentracing.Tracer();
+const tracer = new opentracing.Tracer();
 
-var span = tracer.startSpan('http_request');
-var opts = {
+const span = tracer.startSpan('http_request');
+const opts = {
     host : 'example.com',
     method: 'GET',
     port : '80',
     path: '/',
 };
-http.request(opts, function (res) {
+http.request(opts, res => {
     res.setEncoding('utf8');
-    res.on('error', function (err) {
+    res.on('error', err => {
         // assuming no retries, mark the span as failed
         span.setTag(opentracing.Tags.ERROR, true);
-        span.log({'event': 'error', 'error.object': err);
+        span.log({'event': 'error', 'error.object': err, 'message': err.message, 'stack': err.stack});
         span.finish();
     });
-    res.on('data', function (chunk) {
+    res.on('data', chunk => {
         span.log({'event': 'data_received', 'data': chunk});
```

Comment:
sure, but the example was there before. I only converted the example to use latest JS language features.